ItemDescription3/pageEXT/postedSalesInovicePageExt.50403.PageExt.al

```diff
@@ -0,0 +1,33 @@
+pageextension 50403 apppostedSalesOrderLinePageExt extends "Posted Sales Invoice Subform"
+{
+    layout
+    {
+        addafter("Description 2")
```

Comment:
Here you have logically place, I can say it's ok, make it consistent -)

Reply:
Alright